torch/_torch_docs.py

```diff
@@ -740,7 +740,7 @@ def merge_dicts(*dicts):
 This function checks if :attr:`input` and :attr:`other` satisfy the condition:
 
 .. math::
-    \lvert \text{input} - \text{other} \rvert \leq \texttt{atol} + \texttt{rtol} \times \lvert \text{other} \rvert
+    \lvert \text{input}_i - \text{other}_i \rvert \leq \texttt{atol} + \texttt{rtol} \times \lvert \text{other}_i \rvert \quad \forall i
 """
     + r"""
 elementwise, for all elements of :attr:`input` and :attr:`other`. The behaviour of this function is analogous to
```

Comment:
IMO `\forall` does make sense for `allclose`, but it is already mentioned here in text. We should keep one or the other.

Reply:
@soulitzer now I see what you mean :)